tests/dash/test_dash.py

```diff
@@ -0,0 +1,113 @@
+import logging
+from ipaddress import ip_address
+
+import pytest
+import ptf.packet as scapy
+from ptf.mask import Mask
+import ptf.testutils as testutils
+
+from constants import *
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.disable_loganalyzer
+]
+
+@pytest.fixture
+def inbound_vnet_packets(dash_config_info):
+    inner_packet = testutils.simple_udp_packet(
+        eth_src=dash_config_info[REMOTE_ENI_MAC],
+        eth_dst=dash_config_info[LOCAL_ENI_MAC],
+        ip_src=dash_config_info[REMOTE_CA_IP],
+        ip_dst=dash_config_info[LOCAL_CA_IP],
+    )
+    pa_match_vxlan_packet = testutils.simple_vxlan_packet(
+        eth_src=dash_config_info[REMOTE_PTF_MAC],
+        eth_dst=dash_config_info[DUT_MAC],
+        ip_src=dash_config_info[REMOTE_PA_IP],
+        ip_dst=dash_config_info[LOOPBACK_IP],
+        vxlan_vni=dash_config_info[VNET2_VNI],
+        ip_ttl=64,
+        inner_frame=inner_packet
+    )
+    expected_packet = testutils.simple_vxlan_packet(
+        eth_src=dash_config_info[DUT_MAC],
+        eth_dst=dash_config_info[LOCAL_PTF_MAC],
+        ip_src=dash_config_info[LOOPBACK_IP],
+        ip_dst=dash_config_info[LOCAL_PA_IP],
+        vxlan_vni=dash_config_info[VM_VNI],
+        ip_ttl=0,
+        ip_id=0,
+        inner_frame=inner_packet
+    )
+
+    pa_mismatch_vxlan_packet = pa_match_vxlan_packet.copy()
+    remote_pa_ip = ip_address(unicode(dash_config_info[REMOTE_PA_IP]))
```

Comment:
suggest to modify this line to something like this:
remote_pa_ip = ip_address(dash_config_info[REMOTE_PA_IP])

It will allow to run tests using python3 environment from sonic-mgmt docker, without modification - we will see in python3 error: NameError: name 'unicode' is not defined